draft-jhoyla-tls-key-injection.md

```diff
@@ -83,44 +81,18 @@ extra context implied.
              |
              +-----> Derive-Secret(., "ext binder"
              |                      | "res binder"
-             |                      | "imp ext binder"
-             |                      | "imp res binder", "")
+             |                      | "imp ext binder", "")
              |                     = binder_key
+             v
 ~~~
 
-Using the "imp ext binder" label implies that both parties agree that there is
+Use of the "imp ext binder" label implies that both parties agree that there is
 some context that has been agreed, and that they are using an external PSK.
-Using the "imp res binder" label implies that both parties agree that there is
-some context that has been agreed, and that they are using a resumptionn PSK.
-
-The use of the imp binder derivations means that the PSK ID MUST have an opaque
-context field.
-For example, a PSK with additional data could be constructed as follows.
-
-~~~
-  struct {
-    opaque external_identity<1...2^16-1>;
-    opaque context<0...2^16>;
-  } PSKIDWithAdditionalData
-~~~
```

Comment:
We should include this struct. This should be the "minimum requirement".

Reply:
Hmm... I'm not sure. I agree that there's some minimum requirement, but I think we can state that without giving a structure.